### ISSUE TITLE
 My book essential 1TB/ raw file system, VCD not access

### ISSUE THREAD

**Issue by gregor2507** [andlabs/reallymine#7]
Hi! my problem is that when you connect the WD my book drive via USB to a laptop. it displays the information that, the disk must be formatted, the drive is visible in the system, but you can not do it access the file format is RAW.

I tried to find the mbr by testdisk and DMDE and others software but unfortunately nothing. 

But anyway... 

My situation is a little different than all described here, because the damage has been caused by the user... 
So it was that, after connecting the user disk to a USB port, showed up information from windows the disk must be formatted, and then user accidentally pressing the button formating now......., but the drive quickly disconnected. (I know it's too late)

I forgot to tell first, discs had established a password, but it is known,  so it remains only a matter of encryption SED. 

actually the problem lies in the fact that the disk visible in winhex includes all hex values, so formatting is not removed everything, otherwise you can see there that the record is in the form of encrypted. 

it seems to me that a defective entry responsible for initialization VCD because it does not appear already a message asking you to enter a password.

I think i must try to restore correctly VCD and later will be able to decrypt all files. I tried to install the firmware but unfortunately it did not restore a virtual drive. 

I know everyone of you had a different experience with these hdd and it's different answer sometimes. 

maybe someone is able to do to help? eager to present all the results and values her. 

I have also one more a disk my book essential 1TB WD "green" unfortunately he has an older version of USB bridge. So I think that nothing will change some of the bridge, etc., it seems to me, however, that my USB bridge is not damage and it was not his fault.

I hope someone can advise something....

**Comment by athomic1** [comment]
That situation isn't all that different from others, actually. It's not unusual for a system to fail to read a drive, simply assume it needs formatting, and tell the user just that. The typical user, assuming the system knows what it's about, gives it the okay, and BOOM! There goes your data.

You'd think, by now, Microsoft and Apple would come up with at least some basic built-in tools to detect possible corruption and offer some recovery options, but of course, nobody seems to think about those things. Just not "sexy" enough, I guess. Anyway, that's enough ranting. Just a couple of questions:

You said the drive quickly disconnected. Do you mean that the computer, or the drive itself broke the connection? Or, did the user immediately unplug it, or something like that? I'm not sure that it makes a lot of difference, but if the connection were broken automatically, that might indicate something wrong in the hardware, which could be bad news.

Formatting won't usually wipe all of a drive's data, at least not immediately, unless you set an option to do so. What it usually does is mark all the space free, and wipe out the directory structure. Most of the data will still be there, but eventually lost as new files are written over them. It may be possible to recover some, or even all the data, as long as you don't write anything else to the drive beforehand. I'm not going to try to recommend any tools in this post. I'm sure others here can give you a much better list that I could, and faster.

The biggest problem with these drives is that encryption, and that's only if the enclosure's interface is failing. If it's not, your best bet is to try and extract an image through that interface, without dismantling the drive. BUT, that's assuming the interface WASN'T failing. I'm guessing it was? Let me lay out what I think the situation might be:

1. User connected the intact WD unit to his system, and it wasn't recognized.
2. System reported drive needs formatting, and user told it to go ahead.
3. System started to format the drive, but failed because drive (immediately?) disconnected.

This is probably a best-case scenario. If the drive automatically disconnected, it might have been a failure in the interface, and it's possible no data was actually written. In this case, there's a good chance you can remove the drive from the enclosure, and extract an image that ReallyMine can process. Assuming this IS a decryptable model, the key data will be stored well up toward the end, and highly unlikely to have been overwritten.

Assuming you're ready to dismantle the enclosure (if you haven't done so already), I would recommend going ahead and running ReallyMine against the drive, and see if you can get the image. It should let you know whether it can find a key, and if it does, it should get you a recoverable image without altering the drive contents at all.

Hoping that's clear enough, but feel free to ask further questions if you need more clarification. Not sure about the second drive, but we can maybe tackle that later.

Good luck!

**Comment by gregor2507** [comment]
I'm sorry if I wrote wrong , after connecting the hard drive to USB, that was running Windows 10 was displayed information "You need to format the disk in drive H: before you can use it" 

Then...

It was pressed accidentally "format the disk", and then quickly disconnected from the power supply. 

because he did not know how to stop or disable this process.

**Comment by andlabs** [comment]
The MBR might be trashed but the data could still be there.

**Comment by gregor2507** [comment]
Hello, thanks for the reply, an idea to restore mbr?

**Comment by MrDecay** [comment]
Lets go through scenarios.

1st I'm speaking on the assumption that it is a 3.5 inch drive.

Remove the drive from the case .
And try to run reallymine to check for encryption and decrypt the drive.

Once decrypted . then you can begin rebuilding the mbr/partition table.

All the data recovery software tools need to be working on an unencrypted
drive.

That's like an overview of the process

**Comment by MrDecay** [comment]
Oh yeah testdisk is my tool of choice for mbr checking

**Comment by gregor2507** [comment]
The drive is removed from the case forgot to write earlier,  hard drive is 3.5 inch. The question is about drive, it's has to be connected without USB board? I mean USB bridge. I add photo
[URL=https://imageshack.com/i/plWeps70j][IMG]http://imagizer.imageshack.us/v2/xq90/921/Weps70.jpg[/IMG][/URL]

**Comment by gregor2507** [comment]
Here:
http://imageshack.com/a/img921/6960/Weps70.jpg

**Comment by gregor2507** [comment]
Is JMicron.

**Comment by andlabs** [comment]
Yes, remove the bridge chip and then connect the HDD to your computer via the SATA connector that is now exposed on the drive itself. The easiest way is to sue some sort of SATA-USB cable, but I don't have one to recommend; sorry (though I will say I had better luck with ones sold from a brick and mortar store than ones bought online). If you have a desktop computer you can also open the case and connect the drive there, but you will need an extra SATA cable, and then you should take the drive back out when done.

**Comment by themaddoctor** [comment]
Is this issue still alive?

Gregor, since you have a JMicron, I have a way to retreive the key even if the encryption information was corrupted by the formatting process. Then you could use reallymine to decrypt an image, and do your data recovery.

If you want me to try to find your key, do a hexdump of the start of the disk until you see a repeating pattern of 16 bytes that are not zero. That pattern is likely to be encrypted zeroes, since the beginning of disks often have a lot of them. That pattern and the date of manufacture (from the label on the disk) are what I need.
